NavigationFlowCoordinator/Coordinator.swift

```diff
@@ -8,9 +8,16 @@
 
 import Foundation
 
-@objc public protocol Coordinator: class {
+
+public enum navigationStyle: Int {
```

Comment:
Can we please use UpperCamelCase naming convenction for enums? So in this case that would be NavigationStyle